endpoints/main.go

```diff
@@ -136,20 +165,30 @@ func main() {
 		return
 	}
 	// Data in resources/EndpointSources was taken from https://fhirfetcher.github.io/data.json
-	var listOfEndpoints = fetcher.GetListOfEndpoints(endpointsFile)
+	var listOfEndpoints, err = fetcher.GetListOfEndpoints(endpointsFile)
+	if err != nil {
+		log.Error("Endpoint List Parsing Error: ", err.Error())
+		return
+	}
 	initializeMetrics()
 
 	var queryCount = 0
 	// Infinite query loop
 	for {
 		for _, endpointEntry := range listOfEndpoints.Entries {
 			// TODO: Distribute calls using a worker of some sort so that we are not sending out a million requests at once
-			var url = endpointEntry.FHIRPatientFacingURI
+			var urlString = endpointEntry.FHIRPatientFacingURI
 			var orgName = endpointEntry.OrganizationName
 			// Long polling stats will be queried for every 6 hours
 			//TODO: Config file
 			var longPollingInterval = (queryCount%72 == 0)
-			getHTTPRequestTiming(url, orgName, longPollingInterval)
+			// Specifically query the FHIR endpoint metadata
+			metadataURL, err := url.Parse(urlString)
+			if err != nil {
+				log.Warn("Endpoint URL Parsing Error: ", err.Error())
```

Comment:
if we had a parsing error, don't we need to stop processing this endpoint and move onto the others? should this be an `if/else`?